src/Unicode/UnicodeEncodingForm.dfy

```diff
@@ -109,27 +109,62 @@ abstract module {:options "-functionSyntax:4"} UnicodeEncodingForm {
     }
   }
 
+  /**
+    * Returns the unique partition of the given code unit sequence into minimal well-formed code unit subsequences,
+    * or None if no such partition exists.
+    */
+  function {:tailrecursion} PartitionCodeUnitSequenceCheckedTailRec(s: CodeUnitSeq, resultPrefix: seq<MinimalWellFormedCodeUnitSeq> := [], ghost sPrefix: CodeUnitSeq := []): (maybeParts: Option<seq<MinimalWellFormedCodeUnitSeq>>)
```

Comment:
This function is never used. Do you intend that clients may call it, or is it a leftover from a previous approach?

Reply:
This was my first attempt that broke callers (since they relied on the non-opaque function definition), I'm removing it.